resources/core/charts/k8s-dashboard-proxy/README.md

```diff
@@ -0,0 +1,23 @@
+# Kubernetes Dashboard Proxy
+
+## Overview
+
+The `Kubernetes Dashboard Proxy` is a transparent proxy for the `Kubernetes Dashboard`
+
+Read the [Kubernetes Dashboard Proxy](https://github.com/kyma-project/kyma/blob/master/components/k8s-dashboard-proxy/README.md) document to learn more about Kubernetes Dashboard Proxy in Kyma.
```

Comment:
```suggestion
Read the [Kubernetes Dashboard proxy](https://github.com/kyma-project/kyma/blob/master/components/k8s-dashboard-proxy/README.md) document to learn more about Kubernetes Dashboard proxy in Kyma.
```
Additionally, I would add the following subchapter right below the link:

`##Details `

So above we have the overview and a link to further resources, and below the details of chart configuration.